### PR TITLE
Client board style

### PR DESCRIPTION
**Pull request by clemcx**
css clientBoard

### CODE REVIEW

**Review by RMayeux — APPROVED** [review]
Looking good, approved !

File: templates/client/show.html.twig [hunk 1]
```diff
@@ -1,40 +1,51 @@
...
-    {{ encore_entry_link_tags('board') }}
-    {{ encore_entry_link_tags('app') }}
-{% endblock %}
-    {% block header %}
-<header>
-    <nav>
-        <a class="logoContainer" href=""><img src="http://magouvernance.fr/wp-content/uploads/2019/05/LOGO-TRANSPARENT.png"  alt="logo"/></a>
+    {% block stylesheets %}
+        {{ encore_entry_link_tags('clientBoard') }}
+        {{ encore_entry_link_tags('app') }}
```
Comment (benj-san) [inline templates/client/show.html.twig]: use {{ parent() }} instead of loading app every pages ?

**Review by benj-san — COMMENTED** [review]
Please transform you link tags app into a parent if this file is used all over website.
Beside it everything is clean

File: templates/client/show.html.twig (hunk 1 above)
Comment (RMayeux) [reply templates/client/show.html.twig]: Resolved

File: templates/client/show.html.twig [hunk 2]
```diff
@@ -1,40 +1,51 @@
...
     </div>
     <div id="">
         {% include 'cvAdvisor.html.twig' %}
     </div>
-
-
-    <h1 id="clientName">{{ board.demand.client }}</h1>
-    <h2>Proposition de board:</h2>
+    <h1 id="clientName">Proposition de board</h1>
+    <h2 ></h2>
```
Comment (benj-san) [inline templates/client/show.html.twig]: anything to put inside this tag ?

**Review by benj-san — APPROVED** [review]
Remove the empty tag before merge, everything looks good.

**Pull request merged**